src/openepd/model/pcr.py

```diff
@@ -70,13 +80,29 @@ class Pcr(WithAttachmentsMixin, WithAltIdsMixin, BaseOpenEpdSchema):
         default=None,
         description="Last day on which the document is valid",
     )
+    doc: str | None = pyd.Field(default=None, description="URL to original document, preferably directly to a PDF.")
     parent: Optional["Pcr"] = pyd.Field(
         description="The parent PCR, base PCR, `Part A` PCR",
         default=None,
     )
+    status: PcrStatus | None = pyd.Field(
+        default=None,
+        description="The current release status of this PCR. "
+        "A PCR with valid_until in the past must have status Expired or Sunset; a PCR with valid_until "
+        "more than 5 years in the past must have status Sunset. Compliant systems should automatically "
+        "update these fields within 24 hours.",
+    )
     product_classes: dict[str, str | list[str]] = pyd.Field(
         description="List of classifications, including Masterformat and UNSPC", default_factory=dict
     )
+    applicable_in: list[Annotated[str, pyd.Field(min_length=2, max_length=2)]] | None = pyd.Field(
```

Comment:
EPD.applicable_in and PCR.applicable_in are accepting the same value:
The most recent description:
![image](https://github.com/cchangelabs/openepd/assets/5246563/fe0725b1-e04e-4bee-b1c7-a91bbad410cb)
https://github.com/cchangelabs/openepd-openapi/blob/dc720e754611b39961f0f1e2af50ec42b37d650d/content/schemas/models/EPD/Base.yaml#L143